lms/djangoapps/course_api/blocks/transformers/student_view.py

```diff
@@ -59,14 +59,14 @@ def collect(cls, block_structure):
                 cls.STUDENT_VIEW_MULTI_DEVICE,
                 supports_multi_device,
             )
-            if getattr(block, 'student_view_data', None):
```

Comment:
We can't just remove this - it's a breaking API change and would immediately break all mobile apps that use this API heavily. Or am I missing something?